source/extensions/filters/network/mysql_proxy/mysql_decoder.cc

```diff
@@ -161,8 +161,7 @@ void DecoderImpl::decode(Buffer::Instance& data, uint64_t& offset) {
 }
 
 void DecoderImpl::onData(Buffer::Instance& data) {
-  // TODO(venilnoronha): handle messages over 16 mb. See
-  // https://dev.mysql.com/doc/dev/mysql-server/8.0.2/page_protocol_basic_packets.html#sect_protocol_basic_packets_sending_mt_16mb.
```

Comment:
Thanks!  Looks like this can be updated to https://dev.mysql.com/doc/dev/mysql-server/latest/page_protocol_basic_packets.html
and merged with another PR.